### PR TITLE
Styling

### PR DESCRIPTION
**Pull request by haftamudesta**
this project is styling part of the Bookstore. Bookstore is a simple website that displays a list of books and allows you to:

[ ] Display a list of books
[ ] Add a book
[ ] Remove a selected book

### CODE REVIEW

File: src/components/navbar.jsx [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+              <NavLink to={nlink.path} className={classes.navlinks}>
+                {nlink.Text}
+              </NavLink>
+            </li>
+          ))}
+        </ul>
+        <h1 className={classes.user}>
+          <BiUser />
+        </h1>
+      </nav>
```
Comment (BertrandConxy) [inline src/components/navbar.jsx:36]: - [ ] Good job so far 👏 Please add some space on the right side of the nav bar as well as on the bookList component as on the design given 

Your design
image

Design given
image

File: src/components/Book.jsx [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+        onClick={removeBook}
+        className={classes.remove__btns}
+      >
+        Remove
+      </button>
+      <button type="button" className={classes.edit__btns}>
+        Edit
+      </button>
+    </li>
+  );
```
Comment (BertrandConxy) [inline src/components/Book.jsx]: - [ ] Please work out the font typefaces difference that is in this part of the Book component. The book category should have the same as the buttons below it. Also the should be aligned  on same margin 

your design
image

Given design
image

File: src/components/Book.jsx [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+        type="button"
+        onClick={removeBook}
+        className={classes.remove__btns}
+      >
+        Remove
+      </button>
+      <button type="button" className={classes.edit__btns}>
+        Edit
+      </button>
+    </li>
```
Comment (BertrandConxy) [inline src/components/Book.jsx]: - [ ] There are some components that make up the Book component UI that you have not included even if they should be hard coded. Check out the comparison below 

Given design
image

Your design
image

File: src/components/form.jsx [hunk 4]
```diff
@@ -0,0 +1,46 @@
...
+          {...register('author', { required: true })}
+          placeholder="Author"
+        />
+        <input
+          type="text"
+          className={classes.categories}
+          {...register('category', { required: true })}
+          placeholder="Category"
+        />
+        <input type="submit" value="Submit" className={classes.submit__btn} />
```
Comment (BertrandConxy) [inline src/components/form.jsx:40]: - [ ] Also, please work on this button in order to match the one given on the design. In terms of styles and alignment with neighboring elements. Please consider the comparison below 👇 

Your design
image

Given design
image

**Review by BertrandConxy — CHANGES_REQUESTED** [review]
Hi @haftamudesta, 

Good job so far! 👏 👏 
There are some issues that you still need to work on to go to the next project but you are almost there! 🏃‍♂️ 🏃‍♂️ 


## Highlights 💪 
- No linter errors
- correct flow is used
- good PR


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @BertrandConxy in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/components/Book.jsx [hunk 5]
```diff
@@ -0,0 +1,92 @@
...
+            type="button"
+            onClick={removeBook}
+            className={classes.remove__btns}
+          >
+            Remove
+          </button>
+          <button type="button" className={classes.edit__btns}>
+            Edit
+          </button>
+        </li>
```
Comment (andersonlebon) [inline src/components/Book.jsx]: image

- [ ] Kindly note that you need to use the same font-family as in the design template just like the previous reviewer pointed out

File: src/components/Book.jsx [hunk 6]
```diff
@@ -0,0 +1,92 @@
...
+        <p>
+          Chapter
+          <span>&nbsp; &nbsp;</span>
+          {num2}
+          <span>&nbsp; &nbsp;</span>
+          {getText()}
+        </p>
+        <button type="button" className={classes.update__btn}>
+          update progress
+        </button>
```
Comment (andersonlebon) [inline src/components/Book.jsx:81]: - [ ] Kindly note that this button should look like the button in the template 
Template button
image
Yout button
image

Apply the same styling for you submit button
 
image

**Review by andersonlebon — CHANGES_REQUESTED** [review]
## Hi @haftamudesta 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

almost-there

## Required Changes ♻️
_Check the comments under the review._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question `@andersonlebon` so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by peterrobert — APPROVED** [review]
Hi @haftamudesta, 


### Highlights!

[ ] Good feature branch name ✔️
[ ] Descriptive pull request ✔️


Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread or tag me @peterrobert if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by victorokoroji — APPROVED** [review]
Hi @haftamudesta, 

# ✔️   Status: APPROVED 🏆 🎉 🎊 

You have done a great job on this project, keep it up 👍 🥇. Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
congratulations!  🎉💯🌟 

## Top highlight 🎯

- The Pull Request is following the GitHub Flow ✅ 
- The PR title and description are professional and descriptive ✅ 
- The commit messages are meaningful ✅ 



## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by victorokoroji — APPROVED** [review]
Hi @haftamudesta, 

# ✔️   Status: APPROVED 🏆 🎉 🎊 

You have done a great job on this project, keep it up 👍 🥇. Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
congratulations!  🎉💯🌟 

## Top highlight 🎯

- The Pull Request is following the GitHub Flow ✅ 
- The PR title and description are professional and descriptive ✅ 
- The commit messages are meaningful ✅ 



## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**